pkg/interface/src/views/components/RichText.js

```diff
@@ -37,7 +37,7 @@ const RichText = React.memo(({ disableRemoteContent, ...props }) => (
           return <RemoteContent className="mw-100" url={linkProps.href} />;
         }
 
-        return <Anchor display="inline" target='_blank' rel='noreferrer noopener' borderBottom='1px solid' remoteContentPolicy={remoteContentPolicy} {...linkProps}>{linkProps.children}</Anchor>;
+        return <Anchor display="inline" target='_blank' referrerPolicy='noreferrer noopener' borderBottom='1px solid' remoteContentPolicy={remoteContentPolicy} {...linkProps}>{linkProps.children}</Anchor>;
```

Comment:
Is this right, since it doesn't have dashes like all the others?

Reply:
[Yeah](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel). The web's a weird place.

Reply:
Oh wait, I wonder if there's a referrer policy on this. I shouldn't have changed anchors lol